src/components/Display.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
we are indirectly changing the rules by disabling these rules, let's fix the codebase instead.

kindly use destructuring and add prop-types.